akka-docs/src/main/paradox/typed/cluster-sharding.md

```diff
@@ -338,80 +359,54 @@ to the level below. Only the least recently used entities in the lowest level wi
 higher levels are considered "protected", where entities will have additional opportunities to be accessed before being
 considered for passivation.
 
-To configure a segmented least recently used (SLRU) strategy, with two levels and a protected segment limited to 80% of the total limit:
+To configure a segmented least recently used (SLRU) policy, with two levels and a protected segment limited to 80% of
+the total limit:
 
 @@snip [passivation segmented least recently used](/akka-cluster-sharding/src/test/scala/akka/cluster/sharding/ClusterShardingSettingsSpec.scala) { #passivation-segmented-least-recently-used type=conf }
 
-Or to configure a 4-level segmented least recently used (S4LRU) strategy, with 4 evenly divided levels:
+Or to configure a 4-level segmented least recently used (S4LRU) policy, with 4 evenly divided levels:
 
 @@snip [passivation segmented least recently used](/akka-cluster-sharding/src/test/scala/akka/cluster/sharding/ClusterShardingSettingsSpec.scala) { #passivation-s4-least-recently-used type=conf }
 
-Or using the `withLeastRecentlyUsedPassivationStrategy` method on `ClusterShardingSettings`.
-
-#### Idle timeouts (with least recently used strategy)
+Or using the `withPassivationStrategy` method on `ClusterShardingSettings`.
 
-Passivating idle entities (when they have not received a message for a specified length of time) can also be enabled by configuring the least recently used passivation strategy with an idle timeout:
+### Most recently used policy
 
-@@snip [passivation least recently used with idle](/akka-cluster-sharding/src/test/scala/akka/cluster/sharding/ClusterShardingSettingsSpec.scala) { #passivation-least-recently-used-with-idle type=conf }
-
-Or enable the least recently used passivation strategy with both an active entity limit and an idle timeout using the
-`withLeastRecentlyUsedPassivationStrategy` method on `ClusterShardingSettings`.
-
-### Most recently used passivation strategy
-
-The **most recently used** passivation strategy passivates those entities that have the most recent activity when the
-number of active entities passes a specified limit. The configurable limit is for a whole shard region and is divided
-evenly among the active shards in each region. This strategy is most useful when the older an entity is, the more
-likely that entity will be accessed again; as seen in cyclic access patterns. Configure automatic passivation to use
-the most recently used passivation strategy, and set the limit for active entities in a shard region:
+The **most recently used** policy passivates those entities that have the most recent activity when the number of
+active entities passes a specified limit. The configurable limit is for a whole shard region and is divided evenly
+among the active shards in each region. This policy is most useful when the older an entity is, the more likely that
```

Comment:
It will be good if we add a sub-section on each strategy explaining when they are useful. 
For example, the first paragraph can explain short what it does, then a new paragraph with a sub-title saying something, "when to use it".

On top of that, some visuals to would also be great like we have for streams, but maybe that's too much. And certainly something that can be added later.

Reply:
Ok, I can separate out the how-it-works from the when-it's-useful info, in a separate section for each policy.

I'll think about whether and what kind visuals could be added. Is there anything you have in mind already?

Reply:
I don't think we should be too ambitious with the documentation here.

Maybe we should also evaluate if all strategies are needed. Some are built as combinations of others but maybe not needed for the user to have the possibility to choose among all of them? Reducing the "api" surface and number of choices.

Reply:
Reworked a little, with a separate more advanced section for custom passivation strategies, and added a `when to use` paragraph for each policy.

Reply:
About the graphicals, it was just an idea. Certainly for later, if ever. Agree that it's maybe too ambitious. And the "when to use" section are really clear and doing the job.